packages/blade/src/components/Drawer/types.ts

```diff
@@ -0,0 +1,74 @@
+import type { TestID } from '~utils/types';
+
+type DrawerProps = {
+  /**
+   * Controlled state of drawer open or not
+   */
+  isOpen: boolean;
+
+  /**
+   * Dismiss handler
+   */
+  onDismiss: () => void;
+
+  /**
+   * Show or hide overlay.
+   *
+   * Also decides if clicking outside on overlay closes the drawer or not
+   */
+  showOverlay?: boolean;
+  /**
+   * children node.
+   *
+   * Supports DrawerHeader and DrawerBody
+   */
+  children: React.ReactNode;
+  /**
+   * zIndex property of drawer
+   */
+  zIndex?: number;
+  /**
+   *  Accessibility label for the drawer
+   */
+  accessibilityLabel?: string;
+  /**
+   *  Ref to the element to be focused on opening the drawer.
+   */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  initialFocusRef?: React.MutableRefObject<any>;
+} & TestID;
```

Comment:
testID description is not present in storybook.

Reply:
Resolved all the jsdoc comments in docs PR